frontend/src/component/playground/Playground/PlaygroundResultsTable/FeatureResultInfoPopoverCell/FeatureStrategyList/StrategyList/StrategyItem/PlaygroundStrategyExecution/PlaygroundStrategyExecution.tsx

```diff
@@ -79,5 +82,17 @@ export const StrategyExecution: FC<StrategyExecutionProps> = ({
         ),
     ].filter(Boolean);
 
-    return <ConstraintsList>{items}</ConstraintsList>;
+    const { variant, variants } = strategyResult.result as unknown as Partial<{
+        variant: PlaygroundFeatureSchemaVariant;
+        variants: StrategyVariantSchema[];
+    }>;
+
+    return (
+        <>
+            <ConstraintsList>{items}</ConstraintsList>
+            {variants?.length ? (
+                <RolloutVariants variants={variants} selected={variant?.name} />
```

Comment:
Wasn't this the old component that you removed? Shouldn't this be using the split?

Reply:
`RolloutVariants` is wrapping `VariantsSplitPreview` to match it to constraints list